src/mcp_server_helm/commands/release.py

```
@@ -0,0 +1,320 @@
+    cmd = ["helm", "history", release_name]
+
+    if namespace:
+        cmd.extend(["-n", namespace])
+
+    if max_:
+        cmd.extend(["--max", str(max_)])
+
+    return execute_helm_command(cmd)
+
+
+def helm_status(release_name: str, namespace: Optional[str] = None, revision: Optional[int] = None) -> str:
```

Pylint (3.3.7)

[convention] Line too long (107/100)

(C0301)